app/js/views/components/tableRow.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import jsyaml from 'js-yaml';
+
+import {
+  OverlayTrigger,
+  Button,
+  Popover
+} from 'react-bootstrap';
+
+import EditButton from './editButton.jsx';
+
+export default class TableRow extends React.Component {
+  constructor(params) {
+    super(params);
+  }
+
+  render() {
+    const item = this.props.item;
+    let list = this.props.schema.schema.propertiesOrder.map((key, index) => {
+      const property = this.props.schema.schema.properties[key];
+      const view = property['view'];
+
+      if (view && !view.includes('list')) {
+        return null;
+      }
+      const title = property.title.toLowerCase();
+
+      var urlPattern = /^((http|https):\/\/)/;
+
+      try {
+        if (property.type === 'object' || property.originalType === 'object') {
+          if (item[key] === undefined) {
+            return null;
```

Comment:
false, null and undefined generate no element in React,
whereas  "", 0 and NaN generate <span> elements